views/landing.handlebars

```diff
@@ -0,0 +1,10 @@
+<link rel="stylesheet" href="static/css/landing.css">
+
+<div id="landing">
+  <h1 id="landing-text">
+    Find the best organizations to donate your items to
```

Comment:
Let's also update the landing page unit test to look for this string on the page?